### DISCUSSION TITLE
How to install Orochi on my macOS Sierra 10.12?

### DISCUSSION THREAD

**Discussion by aryanadi-ic** [LDO-CERT/orochi#671] [General]
Hi all..., I just installed orochi on my macOS Sierra 10.12 but I had no idea what next commandline syntax I should execute. I dont hv docker in my macbook, because it's heavy and makes may mac stop running:-( Pls kindly enlight me. Thank you!
![Orochi installation](https://user-images.githubusercontent.com/67795345/170675898-7fe535db-dc64-43e8-8d4b-96f26284238f.png)

**Comment by garanews** [comment]
In order to use OROCHI is mandatory to have docker and docker-compose installed.
